src/diagnostics/diagnostic-codes.md

```diff
@@ -32,7 +32,7 @@ Here we see the highest error code in use is `E0591`, so we _probably_ want
 `E0592`. To be sure, run `rg E0592` and check, you should see no references.
 
 Next, open `src/{crate}/diagnostics.rs` within the crate where you wish to issue
-the error (e.g., `src/librustc_typeck/diagnostics.rs`). Ideally, you will add
+the error (e.g., `compiler/rustc_typeck/src/diagnostics.rs`). Ideally, you will add
```

Comment:
There is no file named `compiler/rustc_typeck/src/diagnostics.rs`. I ran into this while trying to add a new error code: all diagnostics are now in `compiler/rustc_error_codes/src/error_codes.rs`, which has one very long `register_diagnostics!`, and nowhere else adds error codes. `register_long_diagnostics!` also seems to no longer exist, instead that file uses `include_str!("./error_codes/E0XXX.md")`.

Reply:
@jyn514 Ah, I see. Would you mind making a PR for this? I'm a bit low on bandwidth these days.

Reply:
Opened https://github.com/rust-lang/rustc-dev-guide/pull/841. Although both these PRs are now mutually dependent on one another 😅 since this one fixes the broken links for CI.